### DISCUSSION TITLE
Generate tasks dynamically - using either structure mapping or dynamic values

### DISCUSSION THREAD

**Discussion by f-odhiambo** [opensrp/fhircore#436] [Ideas]
- [ ] How do we go about  Structure-mapping and dynamic tasking
- [ ] Build a UI powered by Tasking - Task generation automation

**Comment by pld** [comment]
Questions to answer before implementation:

- [ ] confirm this is to generate tasks dynamically in-app in response to another event (common usage) taking place
- [ ] decide the resource and event (common usage) in-which to encode task generation
- [ ] decide whether to use structure mapping or dynamic values

**Comment by shoaibmushtaq25** [comment]
Need some more details on this issue to have more context to start over it. Can you guys please add them in order to have more clear idea of what should I do in the scope of this issue? @f-odhiambo

**Comment by pld** [comment]
yea agree, the questions in my comment and an implementation plan need to be written before work on this can start, adding blocked label and unassigned for the moment.

In Monday's call we discussed the structure maps are likely not needed for now, more flexibility than we require at the moment. I'm also unclear what here is covered by the work on ResourceMapper and what is not.
